doc/xversionmessage.md

```diff
@@ -92,7 +92,9 @@ during the lifetime of a connection and receival of multiple such
 messages should be considered misbehavior of the remote peer.
 
 After receival of an `xversion` message, a node must answer with an
-empty `verack` message to confirm recept.
+empty `verack` message to confirm recept. With `xversion` enabled the
+handshake should be `version` `xversion` `verack`. Without `xversion`
+enabled the handshake should be `version` `verack`.
 
```

Comment:
How will we know whether to expect an XVersion message... for example... Lets say we have an XVERSION peer we're connecting to,  we receive the Version, but not Xversion...how do we know when to send the VERACK...or in other words we don't really know whether we're supposed to send the VERACK right away or wait for the XVERSION? or are you using some sort of version bit to know what the remote peer is?

Reply:
it is now activated by a service bit. The next free service bit is bit 11 so i have assumed for now that it will be using that bit. i have updated the doc to reflect this

Reply:
I wonder, given the limited number of service bits, whether we should re-purpose bit 3, which was for WITNESS.

Reply:
I dont think it matters. xversion ideally replaces the need to use most version bits anyway.